src/Streamly/Internal/Data/Stream/IsStream/Reduce.hs

```diff
@@ -235,10 +106,40 @@ module Streamly.Internal.Data.Stream.IsStream.Nesting
     -- , slidingSessionBuffer
     -}
 
-    -- * Transform (Nested Containers)
-    -- Nested splitting
+    -- ** Parsing
```

Comment:
You can add a heading similar to the previous two
```
-- * Reduce By Parsers
```

Reply:
Ok for the first one. For moving to the end, the following functions `wordsBy` etc would be implemented as parsers in future.